README.md

```diff
@@ -29,3 +29,6 @@ To build this app required:
 2. APP_ID and APP_HASH changed in tdlibjson_wrapper.pro
 
 
+#### Licenses
+ - sticker-smiled is GPLv3
```

Comment:
Still missing most important: author and original author if that is based on somebody else work.

Reply:
Yes i will..